tests/integration/workspace_access/test_workflows.py

```diff
@@ -52,8 +47,10 @@ def test_running_real_migrate_groups_job(
     installation_ctx.make_grant(ws_group.display_name, 'SELECT', table_info=table)
 
     # TODO: Move `make_secret_scope` and `make_secret_scope_acl` to context like other `make_` methods
-    secret_scope = make_secret_scope()
-    make_secret_scope_acl(scope=secret_scope, principal=ws_group.display_name, permission=AclPermission.WRITE)
+    secret_scope = installation_ctx.make_secret_scope()
+    installation_ctx.make_secret_scope_acl(
+        scope=secret_scope, principal=ws_group.display_name, permission=AclPermission.WRITE
+    )
 
     # TODO: Move `include_object_permissions` to context like other `include_` attributes
```

Comment:
Could you make sure this is happening too? Similar to how the `make_schema` adds the created schema to the database.